src/cli/commands/__tests__/lint.test.ts

```diff
@@ -54,7 +57,49 @@ describe('lint', () => {
       });
   });
 
-  describe('when loading local specification files', () => {
+  describe('when single ruleset option provided', () => {
+    test
+      .stdout()
+      .command(['lint', validSpecPath, '-r', 'non-existent-path'])
+      .exit(2)
+      .it('outputs "does not exist" error');
+
+    test
+      .stdout()
+      .command(['lint', validSpecPath, '-r', invalidRulesetPath])
+      .exit(2)
+      .it('outputs "invalid ruleset" error', ctx => {
+        expect(ctx.stdout).toMatchSnapshot();
```

Comment:
Hey this might just be one persons opinion but I really don’t like match snapshot in 99% of cases. If it’s small can we use inline snapshots or better yet just use expects?

Reply:
done already, marking as resolved